### PR TITLE
Add links to social media

### PR DESCRIPTION
**Pull request by romejkola**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -2,6 +2,29 @@ body {
...
+  color: #e0e0e0;
+  opacity: 30%;
+}
+
+ul li a:hover {
+  color: white;
+  opacity: 100%;
+}
+
+
```
Comment (rpt) [inline styles.css]: Unnecessary line.

File: styles.css [hunk 2]
```diff
@@ -10,16 +33,19 @@ body {
...
+    letter-spacing: 12px;}
+
+.position {
+  position: absolute;
+  text-align: center;
+  width: 95%;
+  padding: 5%;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);}
```
Comment (rpt) [inline styles.css]: Closing bracket should be in the new line.

**Review by rpt — CHANGES_REQUESTED** [review]
Icons are moved a little to the right with some padding.

File: styles.css (hunk 1 above)
Comment (romejkola) [reply styles.css]: Fixed.

File: styles.css (hunk 2 above)
Comment (romejkola) [reply styles.css]: Fixed.

**Review by rpt — APPROVED** [review]
<3

**Pull request merged**